### DISCUSSION TITLE
Table改变列宽后，拖动列位置，列宽改动的值会丢失

### DISCUSSION THREAD

**Discussion by wenlino** [dotnetcore/BootstrapBlazor#3380] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

当列可以改动位置属性和列宽可以改变属性同时使用的时候，改动列宽后，拖动列位置列宽失效。会恢复默认宽度
![动画4](https://github.com/dotnetcore/BootstrapBlazor/assets/58806067/32341817-bff6-4d94-a6f6-ede09f331f0c)

**Comment by bb-auto** [comment]
@wenlino Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@wenlino 请提供复现代码 可能未设置 `ClientTableName` 吧
